src/components/CommentTeaser/docs.md

```diff
@@ -1,122 +1,104 @@
-Provides a preview of a comment, featuring either a title/subtitle or a displayAuthor.
+A teaser of a comment.
+
+Props:
+- `id`: The comment id.
+- `displayAuthor`: The comment's displayAuthor object. If not present, `timeago` will be rendered in the footer.
+- `preview`: The comment's preview object with a `string` (i.e. a short snippet of the comment) and a `more` boolean.
+- `highlights`: An optional array containing highlighted strings for search results. If present, it will trump `preview`.
+- `discussion`: The comment's discussion object. Used to extract title and link properties.
+- `tags`: An optional array of tags. Currently only the first tag is used, but that might change in the future.
+- `parentIds`: The comment's array of parent ids. Used to determine whether it's a comment or a reply.
+- `createdAt`: The comment's creation timestamp.
+- `timeago`: A function that renders a human-readable version of `createdAt`.
+- `Link`: A link wrapper component.
+- `onClick`: An optional click handler.
```

Comment:
Drop this unless really needed. One should always use a `Link` component with attaches a `onClick` and `href`.

Document that `Link` always gets a `commentId: string` `discussion: shapeOf({title: string, id: string})` and `passHref: bool` prop. See front template as an documentation example:
https://styleguide.republik.ch/templates/front